test/browser_tests/page_objects/page_dbwu-small-businesses.js

```diff
@@ -26,7 +26,7 @@ function SmallBusinessess() {
 
   this.smallBusinessLinks = this.smallBusinessInfo.all( by.css( 'a' ) );
 
-  this.awardsTable = element( by.css( '.content_main .simple-table' ) );
+  this.awardsTable = element( by.css( '.content_main .o-table' ) );
```

Comment:
The [awards table test](https://github.com/cfpb/cfgov-refresh/blob/3d0f5c6fec63ee0a9b01941253a1fbfa81a6f61f/test/browser_tests/spec_suites/content/dbwu-small-businesses.js#L41-L44) is failing because it still has the `simple-table` class. Was it Wordpress content?